package.json

```diff
@@ -1,22 +1,28 @@
 {
   "name": "@testing-library/vue",
-  "version": "2.0.0",
+  "version": "0.0.0-semantically-released",
   "description": "Simple and complete Vue DOM testing utilities that encourage good testing practices.",
   "main": "dist/vue-testing-library.js",
```

Comment:
I believe this will need to be changed

Reply:
Oh, how come? I was expecting Travis + semantic release will set the version right before it releases.

Actually, I wanted to ask you something. I saw that `semantic-release` uses git commit messages to compute the required version bump. I've noticed, however, that you are not enforcing any commit structure in RTL, for instance ([example](https://github.com/testing-library/react-testing-library/pull/427)). How does that work?

Reply:
Oh, sorry Kent, I thought you mean the `version` field, now I see you added your comment two lines below. I double checked, and since the file is called `src/vue-testing-library.js`, the generated dist/ folder keeps the same naming.

Reply:
When I merge pill requests I use the "Squash and merge" option which allows me to write my own commit message. Much less of a burden on contributors and I don't need commitizen.